platform/o.n.bootstrap/test/unit/src/org/netbeans/ModuleManagerTest.java

```diff
@@ -2911,15 +2918,15 @@ public void testAutoloadFragmentEnablesHostAndPeers() throws Exception {
         c.checkHostAndOtherFragmentsLoaded(c.fragmentAutoload);
     }
     
-    public void testAutoloadHostEnablesAllFragments() throws Exception {
+    public void testAutoloadHostEnablesEagerFragments() throws Exception {
         ModsCreator c = new ModsCreator();
         createTestJAR(data, jars, "client-module-depend-host", "client-module");
         c.loadModules();
         
         Module client = c.mgr.create(new File(jars, "client-module-depend-host.jar"), null, false, false, false);
         c.mgr.enable(client);
 
-        c.checkHostAndOtherFragmentsLoaded(c.host);
+        c.checkHostAndOtherFragmentsLoaded();
```

Comment:
This is the real change in behaviour: some fragments are no longer eagerly loaded (if they are not eager).